dom0-updates/qubes-dom0-update

```diff
@@ -211,19 +206,21 @@ qvm-run --nogui -q "$UPDATEVM" 'rm -rf /var/lib/qubes/dom0-updates/etc' || exit
 tar c /var/lib/rpm /etc/yum.repos.d /etc/yum.conf /etc/dnf/dnf.conf 2>/dev/null | \
    qvm-run --nogui -q --pass-io "$UPDATEVM" 'LC_MESSAGES=C tar x -C /var/lib/qubes/dom0-updates 2>&1 | grep -v -E "s in the future"'
 
-CMD="/usr/lib/qubes/qubes-download-dom0-updates.sh --doit --nogui $(escape_args "${ALL_OPTS[@]}")"
-# Use 'script' to fake a terminal, so that we are shown sync and download
-# progress indicators. However, do it only if we are running in terminal
-# ourselves.
-if [ -t 1 ] && [ -t 2 ]; then
-    CMD="script --quiet --return --command '$CMD' /dev/null"
+CMD="/usr/lib/qubes/qubes-download-dom0-updates.sh --doit --nogui"
+
+# If we are running in a terminal, run the command in its own terminal in the UpdateVM.
+if [ -t 1 ] || [ -t 2 ]; then
+    CMD="xterm -e $CMD"
```

Comment:
This breaks running qubes-dom0-update without GUI fully working - which is bad, because you may want to use it to download a fix to get GUI back. Also, `qvm-run --nogui` is used, which doesn't guarantee you can rely on GUI (and also - if for some reason gui-agent fails to start, the command will hang)
Note, qvm-run does remove terminal control characters when connected to a terminal.

Generally, I think this change significantly reduce reliability of `qubes-dom0-update` tool for very little gain.
Other changes in this PR are useful, though.

Reply:
What about having a flag to control this?